cmd/read/read.go

```diff
@@ -30,6 +32,10 @@ func New(cloud *common.Cloud) *cobra.Command {
 	}
 
 	cmd.Flags().IntVar(&o.Parallelism, "parallelism", 1, "parallelism")
+	cmd.Flags().BoolVar(&o.Status, "status", false, "Read status")
+	cmd.Flags().BoolVar(&o.Events, "events", false, "Read events")
```

Comment:
What's the difference between events and logs?

Reply:
Events are “scaling events” like “requested 3 instances, there is not enough capacity to satisfy the request”; logs are the output of the `script` provided by the user.

Reply:
Moved events to `DEBUG` level logs in `--status`

Reply:
@0x2b3bfa0 
so, may I suggest:
`o.Events` -> `o.ScalingEvents` or `o.InfrastructureEvents`?

Reply:
`o.Events` doesn't exist anymore, as it was “merged” with `o.Status` 🤔

Reply:
Would you suggest adding again a third option (e.g. `--scaling-events`) to print those events to standard output?